tests/AutoReview/ProjectCodeTest.php

```diff
@@ -88,6 +88,10 @@
 
     public function test_infection_bin_is_executable()
     {
+        if (DIRECTORY_SEPARATOR === '\\') {
```

Comment:
we have such check for windows:

```
if (stripos(PHP_OS, 'WIN') === 0) {
```

would it be consistent to use the same? not a big deal though